DiscordChatExporter.Core/Exporting/ExportRequest.cs

```diff
@@ -122,4 +139,20 @@ public partial record ExportRequest
         // Output is a file
         return actualOutputPath;
     }
+
+    private static string GetOutputAssetsDirPath(
+        string basePath,
+        Guild guild,
+        Channel channel,
+        string? assetsPath,
+        Snowflake? after = null,
+        Snowflake? before = null)
+    {
+        if (assetsPath is null)
+            return $"{basePath}_Files{Path.DirectorySeparatorChar}";
+
+        var actualAssetsPath = FormatPath(assetsPath, guild, channel, after, before);
+
+        return actualAssetsPath;
+    }
 }
```

Comment:
Can we pull the `assetsPath is null` case outside in the property evaluator so that it becomes:

```csharp
public string OutputAssetsDirPath => _outputAssetsDirPath ??= (
    AssetsPath is not null
        ? GetOutputAssetsDirPath(
		        Guild,
		        Channel,
		        AssetsPath,
		        After,
		        Before
    		  )
        : $"{OutputBaseFilePath}_Files{Path.DirectorySeparatorChar}"
);
```

Reply:
Should we even bother with having a `GetOutputAssetsDirPath`? It'd be identical to `FormatPath`/`EvaluateTemplateTokens` with this change so we could probably just call the latter directly.